README.md

```diff
@@ -126,6 +126,12 @@ docker run -d --name seafile \
 
 If you are one of the early users who use the `launcher` script, you should refer to [upgrade from old format](https://github.com/haiwen/seafile-docker/upgrade_from_old_format.md) document.
 
+### Garbage Collection
+
+When files are deleted, the blocks comprising those files are marked as unused but not immediately removed. To remove them, Seafile requires a ['garbage collection'](https://manual.seafile.com/maintain/seafile_gc.html) process to be run.
```

Comment:
Technically they are not marked as deleted. The garbage collector tries to find out whether they are still used or not. It is only guaranteed that no required blocks are removed. It is not guaranteed that all blocks that could be removed are actually getting removed (some might remain and be deleted in another run).